common/notification/actions.ts

```diff
@@ -226,6 +233,13 @@ const _notificationActions = {
             },
         },
     },
+    instructor_course_ended: {
+        description: 'Instructor / Course ended',
+        sampleContext: {
+            uniqueId: '1',
```

Comment:
I think the uniqueId can be omitted here, as it is a Notification System property.